app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/LoadingLocationXmlWriter.scala

```diff
@@ -25,7 +25,7 @@ object LoadingLocationXmlWriter {
   implicit val loadingLocationXmlWriter: XmlWriter[LoadingLocation] = new XmlWriter[LoadingLocation] {
     override def toXmlOption(value: LoadingLocation): Option[Elem] = {
       val id: Node = element("ID", value.id)
-      Some(<LoadingLocation>{id}</LoadingLocation>)
+      Option(<LoadingLocation>{id}</LoadingLocation>)
```

Comment:
why this is changed, is it necessary as null check not required here?

Reply:
We wanted to keep the writers consistent. Most use the `Option` factory to create the optional return type.

Reply:
ok, just don't see the point of having extra null check from `Option.apply()` here